README.md

```diff
@@ -35,7 +35,18 @@ On every Pull Request a GitHub action will trigger `make` to check if everything
 
 Once your PR is merged, an additional action will run make and check those changes in.
 
+## Development
+
+In order to develop the repo follow these steps to get an env started:
+
+1. run `make && make test` to test build and deploy
+2. change [Makefile](./Makefile) and [config/manager/manager.yaml](config/manager/manager.yaml) to point to the repo you wish to use
+3. build and deploy with `make docker-build && make docker-push`
+4. use `make deploy` to deploy your operator on a cluster you are logged into
+5. check logs with `kubectl logs -n openshift-logging deploy/route-monitor-operator-controller-manager -c manager`
```

Comment:
That can safely be `oc` instead of `kubectl`

Reply:
yeah, wanted the docs to be neutral, but will rename